From linux-ext4-owner@vger.kernel.org Sun Feb 17 01:18:04 2008
Return-Path: <linux-ext4-owner@vger.kernel.org>
Received: from imap.linux.ibm.com ([unix socket]) by imap.linux.ibm.com
	(Cyrus v2.3.7-Invoca-RPM-2.3.7-7) with LMTPA; Sun, 17 Feb 2008 01:18:04
	-0500
X-Sieve: CMU Sieve 2.3
Received: by imap.linux.ibm.com (Postfix, from userid 101) id 2640A191015F;
	Sun, 17 Feb 2008 01:18:04 -0500 (EST)
X-Spam-TestScore: none
X-Spam-TokenSummary: Bayes not run.
X-Spam-Checker-Version: SpamAssassin 3.1.7 (2006-10-05) on
	imap.linux.ibm.com
X-Spam-Level: 
X-Spam-Status: No, score=0.0 required=5.0 tests=none autolearn=disabled 
	version=3.1.7
X-Spam-Relay-Country: US ** US US US ** US US US XX JP
Received: from smtp.linux.ibm.com (smtp.linux.ibm.com [9.26.4.197]) by
	imap.linux.ibm.com (Postfix) with ESMTP id AE0AC1910121 for
	<cmm@imap.linux.ibm.com>; Sun, 17 Feb 2008 01:18:00 -0500 (EST)
Received: from localhost (localhost.localdomain [127.0.0.1]) by
	smtp.linux.ibm.com (Postfix) with ESMTP id 67DE8C078 for
	<cmm@linux.ibm.com>; Sun, 17 Feb 2008 01:18:00 -0500 (EST)
X-Virus-Scanned: amavisd-new at linux.ibm.com
Received: from BLDGATE.BOULDER.IBM.COM (bldgate.boulder.ibm.com
	[9.17.210.138]) by smtp.linux.ibm.com (Postfix) with ESMTP id 1E3DBC03E for
	<cmm@linux.ibm.com>; Sun, 17 Feb 2008 01:17:59 -0500 (EST)
Received:  by BLDGATE.BOULDER.IBM.COM (IBM VM SMTP Level 520) via spool
	with SMTP id 7217 ; Sat, 16 Feb 2008 23:17:59 MST
Received: by bldgate.vnet.ibm.com (xagent2 5.9.5) via xagsmtp3 with spool
	id 7503 for cmm@linux.vnet.ibm.com; Sat, 16 Feb 2008 23:17:59 -0700 (MST)
Received: from d03relay01.boulder.ibm.com [9.17.195.226] by
	BLDGATE.BOULDER.IBM.COM (IBM VM SMTP Level 520) via TCP with ESMTP ; Sat,
	16 Feb 2008 23:17:59 MST
Received: from d03av01.boulder.ibm.com (d03av01.boulder.ibm.com
	[9.17.195.167]) by d03relay01.boulder.ibm.com (8.13.8/8.13.8/NCO v8.7) with
	ESMTP id m1H6HxQ0105618; Sat, 16 Feb 2008 23:17:59 -0700
Received: from d03av01.boulder.ibm.com (loopback [127.0.0.1]) by
	d03av01.boulder.ibm.com (8.12.11.20060308/8.13.3) with ESMTP id
	m1H6HwEs014086; Sat, 16 Feb 2008 23:17:58 -0700
Received: from d03as01.boulder.ibm.com (d03as01 [9.17.195.252]) by
	d03av01.boulder.ibm.com (8.12.11.20060308/8.12.11) with ESMTP id
	m1H6HwRL014081 (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256
	verify=OK); Sat, 16 Feb 2008 23:17:58 -0700
Received: from e36.co.us.ibm.com ([9.17.249.46]) by d03as01.boulder.ibm.com
	(8.12.11.20060308/8.12.11) with ESMTP id m1H6HvBD011383
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=OK); Sat, 16
	Feb 2008 23:17:57 -0700
Received: from vger.kernel.org (vger.kernel.org [209.132.176.167]) by
	e36.co.us.ibm.com (8.13.8/8.13.8) with ESMTP id m1H62fYC027136; Sun, 17 Feb
	2008 01:17:57 -0500
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand id
	S1752505AbYBQGR4 (ORCPT <rfc822;jrs@us.ibm.com> + 2 others); Sun, 17 Feb
	2008 01:17:56 -0500
Received: (majordomo@vger.kernel.org) by vger.kernel.org id
	S1752515AbYBQGR4 (ORCPT <rfc822;linux-ext4-outgoing>); Sun, 17 Feb 2008
	01:17:56 -0500
Received: from wa-out-1112.google.com ([209.85.146.182]:3255 "EHLO
	wa-out-1112.google.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org with
	ESMTP id S1752505AbYBQGRz (ORCPT <rfc822;linux-ext4@vger.kernel.org>); Sun,
	17 Feb 2008 01:17:55 -0500
Received: by wa-out-1112.google.com with SMTP id v27so2077834wah.23 for
	<linux-ext4@vger.kernel.org>; Sat, 16 Feb 2008 22:17:55 -0800 (PST)
DKIM-Signature: 	v=1; a=rsa-sha256; c=relaxed/relaxed;        d=gmail.com;
	s=gamma;
	h=domainkey-signature:received:received:date:from:to:cc:subject:message-id:mime-version:content-type:content-disposition:user-agent;
	       bh=BXTTq54KZGMxBGiZxaAOwntdTuCcn6ROqxbSR+nJDQM=;       
	b=MaVDL5z3KHOCrJQ9u7fxpGNuorjk00W8f+RsK9zXvQA6gic0SSzX4MPMaaqdXp0qrTzH3l3P49NYehD4dGqCHPezJoORHqVHfIXRX9fFBfZx2MCFUIkI2LABf0NmyZ2b8TY4WlH4l0GQxkLNu7yO/ETTTSig8uhFykL3L7bmZYw=
DomainKey-Signature: a=rsa-sha1; c=nofws;        d=gmail.com; s=gamma;
	h=date:from:to:cc:subject:message-id:mime-version:content-type:content-disposition:user-agent;
	      
	b=CNhwDbxo3l3oIdhbfk2pJmvqdA8zYTd3CFOUO/qAojtAbV3/CzewD5NBidXXK3lLS6yh+qsK/czvoJ9twQv8YBoZ93ZD2l9O+L/6fKMfLgb/B9I7BC+0mis1quXpHZ77hMd9s/3wwJlPI4r1RlqoeYLHvt9hX/QDo9UPM0m0a0A=
Received: by 10.114.39.16 with SMTP id m16mr1134277wam.146.1203229075432;
	Sat, 16 Feb 2008 22:17:55 -0800 (PST)
Received: from @ ( [222.148.118.78]) by mx.google.com with ESMTPS id
	m26sm12784959pof.13.2008.02.16.22.17.53 (version=SSLv3 cipher=OTHER); Sat,
	16 Feb 2008 22:17:54 -0800 (PST)
Date: 	Sun, 17 Feb 2008 15:11:38 +0900
From: Akinobu Mita <akinobu.mita@gmail.com>
To: linux-ext4@vger.kernel.org
Cc: Andrew Morton <akpm@linux-foundation.org>,        Stephen Tweedie <sct@redhat.com>, adilger@clusterfs.com,        Mingming Cao <cmm@us.ibm.com>, Theodore Tso <tytso@MIT.EDU>
Subject: [PATCH 2/4] ext4: add missing ext4_journal_stop()
Message-ID: <20080217061137.GI3390@APFDCB5C>
MIME-Version: 1.0
Content-Type: text/plain; charset=iso-2022-jp
Content-Disposition: inline
User-Agent: Mutt/1.5.17 (2007-11-01)
Sender: linux-ext4-owner@vger.kernel.org
Precedence: bulk
List-ID: <linux-ext4.vger.kernel.org>
X-Mailing-List: 	linux-ext4@vger.kernel.org
X-Xagent-From: akinobu.mita@gmail.com
X-Xagent-To: cmm@linux.vnet.ibm.com
X-Xagent-Gateway: bldgate.vnet.ibm.com (XAGENTU at BLDGATE)
X-Evolution-Source: imap://cmm@imap.linux.ibm.com/
Content-Transfer-Encoding: 8bit

Add missing ext4_journal_stop() in error handling.

Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
Cc: Stephen Tweedie <sct@redhat.com>
Cc: adilger@clusterfs.com
Cc: Andrew Morton <akpm@linux-foundation.org>
Cc: Mingming Cao <cmm@us.ibm.com>
Cc: Theodore Tso <tytso@MIT.EDU>
---
 fs/ext4/resize.c |    1 +
 1 file changed, 1 insertion(+)

Index: 2.6-rc/fs/ext4/resize.c
===================================================================
--- 2.6-rc.orig/fs/ext4/resize.c
+++ 2.6-rc/fs/ext4/resize.c
@@ -1037,6 +1037,7 @@ int ext4_group_extend(struct super_block
 		ext4_warning(sb, __FUNCTION__,
 			     "multiple resizers run on filesystem!");
 		unlock_super(sb);
+		ext4_journal_stop(handle);
 		err = -EBUSY;
 		goto exit_put;
 	}
-
To unsubscribe from this list: send the line "unsubscribe linux-ext4" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
